samples/bookstore.yaml

```diff
@@ -0,0 +1,55 @@
+name: Autogenerated
```

Comment:
It's interesting that the pattern file can be autogenerated. What's the difference between `Autogenerated` and [`MesheryGeneratedPatternFile`](https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/samples/bookInfoPattern.yaml#L1) ?

Reply:
I am pretty sure this is meshery generated, so not really sure about the difference